assets/wizards/popups/components/popup-popover/index.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * Popup Action Card
+ */
+
+/**
+ * WordPress dependencies.
+ */
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import { decodeEntities } from '@wordpress/html-entities';
+import { MenuItem } from '@wordpress/components';
+import { ESCAPE } from '@wordpress/keycodes';
+
+/**
+ * Material UI dependencies.
+ */
+import EditIcon from '@material-ui/icons/Edit';
+import DeleteIcon from '@material-ui/icons/Delete';
+import PreviewIcon from '@material-ui/icons/Visibility';
+import FrequencyIcon from '@material-ui/icons/Today';
+import TestIcon from '@material-ui/icons/BugReport';
+import SitewideDefaultIcon from '@material-ui/icons/Public';
+
+/**
+ * Internal dependencies.
+ */
+import { Popover, SelectControl, ToggleControl } from '../../../../components/src';
+import './style.scss';
+
+const frequencyMap = {
+	never: __( 'Never', 'newspack' ),
+	once: __( 'Once', 'newspack' ),
+	daily: __( 'Once a day', 'newspack' ),
+	always: __( 'Every page', 'newspack' ),
+};
+
+const frequenciesForPopup = ( { options } ) => {
+	const { placement } = options;
+	return Object.keys( frequencyMap )
+		.filter( key => ! ( 'always' === key && 'inline' !== placement ) )
+		.map( key => ( { label: frequencyMap[ key ], value: key } ) );
+};
+
+class PopupPopover extends Component {
+	/**
+	 * Render.
+	 */
+	render = () => {
+		const {
+			deletePopup,
+			popup,
+			previewPopup,
+			setSitewideDefaultPopup,
+			onFocusOutside,
+			updatePopup,
+		} = this.props;
+		const { id, sitewide_default: sitewideDefault, edit_link: editLink, options } = popup;
+		const { frequency, placement } = options;
+		return (
+			<Popover
+				position="bottom left"
+				onFocusOutside={ onFocusOutside }
+				onKeyDown={ event => ESCAPE === event.keyCode && onFocusOutside() }
+			>
+				{ 'inline' !== placement && (
+					<MenuItem
+						onClick={ () => {
+							setSitewideDefaultPopup( id, ! sitewideDefault );
+							onFocusOutside();
+						} }
+						icon={ <SitewideDefaultIcon /> }
+						className="newspack-button"
+					>
+						{ __( 'Sitewide default', 'newspack' ) }
+						<ToggleControl
+							className="newspack-popup-action-card-popover-control"
+							checked={ sitewideDefault }
+							onChange={ value => {
+								setSitewideDefaultPopup( id, value );
```

Comment:
This cause the request to be fired twice and lead to a possible race condition. The whole `onChange` prop can be skipped, as the `onClick` on `MenuItem` will capture clicks on the `ToggleControl `. 
Same story with test mode setting toggle below.

Reply:
Good catch. https://github.com/Automattic/newspack-plugin/pull/515/commits/1aa547640224a292fe21c2426c30f00d73c52042